controllers/validation/accountpool_validation_controller.go

```diff
@@ -0,0 +1,189 @@
+package validation
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"strconv"
+
+	"gopkg.in/yaml.v2"
+
+	awsv1alpha1 "github.com/openshift/aws-account-operator/api/v1alpha1"
+	"github.com/openshift/aws-account-operator/pkg/awsclient"
+
+	"github.com/go-logr/logr"
+	//"github.com/openshift/aws-account-operator/pkg/totalaccountwatcher"
+	"github.com/openshift/aws-account-operator/pkg/utils"
+	"github.com/openshift/aws-account-operator/test/fixtures"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var accountpoollog = logf.Log.WithName("controller_accountpoolvalidation")
+
+const (
+	accountpoolcontrollerName  = "accountpoolvalidation"
+	AccountPendingVerification = "PendingVerification"
+)
+
+type AccountPoolValidationReconciler struct {
+	Client           client.Client
+	Scheme           *runtime.Scheme
+	awsClientBuilder awsclient.IBuilder
+}
+
+func (r *AccountPoolValidationReconciler) accountSpecUpdate(reqLogger logr.Logger, account *awsv1alpha1.Account) error {
```

Comment:
We have this defined in another package, just pull it in from there. If it's private, let's look to make it public